src/List.test.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import List from './List';
+
+/**
+ * 테스트 목록
+ *
+ * list가 아무것도 없을때 "할 일이 없어요!"가 보이는지
+ * List가 있을때 task가 화면에 보이는지
+ */
+
+describe('List', () => { // 테스트 하려는 대상
+  const handleClickDelete = jest.fn();
+
+  const renderList = ({ tasks }) => render(
+    <List tasks={tasks} onClickDelete={handleClickDelete} />,
+  );
```

Comment:
```suggestion
  const renderList = ({ tasks }) => render((
    <List 
      tasks={tasks}
      onClickDelete={handleClickDelete}
     />,
  ));
```
이렇게 formatting 해봤는데 비교했을 때 어떠하신가요?

Reply:
formatting 후가 훨씬 가독성이 좋은것 같아요 다른 테스트에도 모두 적용했습니다.